lib/system/plugin-vue-loader.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Single File Vue Component Loader.
+ */
+/* global System, URL, fetch */
+exports.fetch = async (url) => {
+  const {vue} = System.getConfig().meta['*.vue'];
+  const path = new URL(url.name).pathname;
+  const response = await fetch(
+    vue.compiler + '?component=' + encodeURIComponent(path));
+  if(response.status !== 200) {
+    const error = await response.json();
```

Comment:
Not sure how comprehensive this non-200 path needs to be.  Could get errors that throw while trying to create the message to throw.